code/data_structures/src/tree/tree/trie/trie.c

```diff
@@ -7,63 +7,68 @@
 #define ALPHABET_SIZE (26)
 #define INDEX(c) ((int)c - (int)'a')
 #define FREE(p) \
-    free(p);    \
-    p = NULL;
- 
+free(p);    \
+p = NULL;
+
```

Comment:
The indentation here was probably intentional, since the statements are part of the `FREE` macro. I don't see what removing the indent adds.

Reply:
Thanks for noting that. I'll fix it.